### ISSUE TITLE
Unable to restore any file

### ISSUE THREAD

**Issue by tobecwb** [restic/restic#1904]
Output of `restic version`
--------------------------
restic 0.9.1 compiled with go1.10.3 on windows/amd64

How did you run restic exactly?
-------------------------------
restic restore --target=e:\temp b732fdec

What backend/server/service did you use to store the repository?
----------------------------------------------------------------
local storage

Expected behavior
-----------------
restore all files

Actual behavior
---------------
repository db146c81 opened successfully, password is correct
restoring <Snapshot b732fdec of [\\\\?\\e:\\ARQUIVOS\\PUBLICO] at 2018-07-23 11:00:01.8815664 -0300 -03 by INTRANET\Administrador@PROJE-ARCGIS03> to e:/temp
ignoring error for \\: node has invalid name
There were 1 errors

**Comment by tobecwb** [comment]
the problem was that i'm trying to restore using a Windows OS. If I use the same command in a Linux system, everything works correctly.
The backup was made using Windows, but I have to use Linux to restore.

**Comment by fd0** [comment]
Thanks for the report! In order to find out what's going on here, can you please paste the output of `restic ls -l latest`, so we can see the directory structure in the repo? Thanks!

**Comment by tobecwb** [comment]
> -rw-rw-rw-     0     0 387273 2015-01-29 23:35:44 /\\?\e:/ARQUIVOS/TECNICO/.....zip
> -rw-rw-rw-     0     0    414 2015-01-29 23:35:44 /\\?\e:/ARQUIVOS/TECNICO/.....log
> -rw-rw-rw-     0     0     62 2015-01-29 23:36:15 /\\?\e:/ARQUIVOS/TECNICO/....dwl
> -rw-rw-rw-     0     0    212 2015-01-29 23:35:38 /\\?\e:/ARQUIVOS/TECNICO/....dwl2
> -rw-rw-rw-     0     0  35328 2015-01-29 23:35:45 /\\?\e:/ARQUIVOS/TECNICO/.....db
> drwxrwxrwx     0     0      0 2015-01-29 23:36:14 /\\?\e:/ARQUIVOS/TECNICO/....
> drwxrwxrwx     0     0      0 2015-01-29 23:36:12 /\\?\e:/ARQUIVOS/TECNICO/....
> drwxrwxrwx     0     0      0 2015-01-29 23:35:45 /\\?\e:/ARQUIVOS/TECNICO/....

changed the filenames... this backup has more than 10TB with millions of files.
but take attention to `/\\?\e:/` on path of each file and/or dir

**Comment by fd0** [comment]
Thanks! This looks like another corner case for Windows which I did not know about ;)

**Comment by ProactiveServices** [comment]
Another output of `ls` when restic backs up from a UNC path via `restic backup \\quadra\test`:

```
snapshot 53662d2d of [\\quadra\test\subdir] at 2018-07-30 15:41:47.4426736 +0100 BST):
drwxrwxrwx     0     0      0 2018-06-27 03:51:29 /\\quadra\test
drwxrwxrwx     0     0      0 2018-07-30 15:37:01 /\\quadra\test/subdir
-rw-rw-rw-     0     0 1677824 2018-07-10 21:01:53 /\\quadra\test/subdir/file.txt
```
Note that GitHub's markdown may be swallowing double-backslashes unless the example is blocked by three backticks

**Comment by rawtaz** [comment]
Can someone look into fixing this? Shouldn't be too hard I guess.

**Comment by MichaelEischer** [comment]
Duplicate of #2807.
